datafusion/substrait/src/producer.rs

```diff
@@ -74,6 +74,7 @@ pub fn to_substrait_plan(plan: &LogicalPlan) -> Result<Box<Plan>> {
 
     // Return parsed plan
     Ok(Box::new(Plan {
+        version: None, // TODO not sure what version should go here?
```

Comment:
I don't know either so more of a question. Is this intended to be the substrait version?? So in this context would be `0.3` you think?

Reply:
https://github.com/substrait-io/substrait-rs/pull/46 adds a [`version`](https://github.com/substrait-io/substrait-rs/pull/46/files#diff-7bb693ee02aca4d9c13232347bb4e13cfad03f9573a547515b43800096e5ce7fR17) function that returns the version of Substrait used to build the crate. That could be used here. Or something like [`version_with_producer("datafusion")`](https://github.com/substrait-io/substrait-rs/pull/46/files#diff-7bb693ee02aca4d9c13232347bb4e13cfad03f9573a547515b43800096e5ce7fR33). Please note that the version field of a plan is required.

Reply:
I also added a link to the issue in the code comment